### PR TITLE
Considerations for authentication and authorization technology

### PR DESCRIPTION
**Pull request by LauraDamianTNA**

### CODE REVIEW

**Comment by suzannehamilton** [comment]
GitHub's not rendering the table properly - I think you just need to add a .md file extension.

File: technology-considerations/Considerations for authentication and authorization technology [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+| Supports   GDPR                           | We need work to prove                                            | 3     | Established                                                  | 5     | Established                                                    | 5     | We need work to prove                                                                                                                   | 3     |
+| Supports   GDS                            | On balance                                                       | 3     | On balance                                                   | 3     | On balance                                                     | 3     | On balance                                                                                                                              | 3     |
```
Comment (suzannehamilton) [inline technology-considerations/Considerations for authentication and authorization technology]: Did you find the GDS article you were using for this part of the evaluation? It would be worth adding a link. Or if "supports GDS" just means "is open source", maybe we should reword it to that?

File: technology-considerations/Considerations for authentication and authorization technology [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+| Supports   WCAG                           | Can build what's needed with   time                              | 5     | Lack of customisation                                        | 3     | Limited customisation                                          | 3     | Complete customisation including   existing themes for the GDS design system                                                            | 5     |
+| Total   Scores                            | Silhouette with own                                              | 61    | Cognito                                                      | 66    | 0Auth                                                          | 63    | Keycloak                                                                                                                                | 68    |
```
Comment (suzannehamilton) [inline technology-considerations/Considerations for authentication and authorization technology]: Could we add a line to say that we were just using the scores for information, not as the definitive decision?

I think it would also be helpful for us when we're looking back at this if we added a few lines summarising the main factors that ended up influencing the decision.

File: technology-considerations/Considerations for authentication and authorization technology (hunk 1 above)
Comment (LauraDamianTNA) [reply technology-considerations/Considerations for authentication and authorization technology]: No, GDS have revised this and they talk about the total cost of ownership, rather than just recommending open source - I included the link.

File: technology-considerations/Considerations for authentication and authorization technology (hunk 2 above)
Comment (LauraDamianTNA) [reply technology-considerations/Considerations for authentication and authorization technology]: Added a few lines.

**Comment by LauraDamianTNA** [comment]
I have put the matrix and the comments in the branch that Sam has started and I'll close this.
